stable/redis-sentinel/templates/sentinel-statefulset.yaml

```diff
@@ -0,0 +1,144 @@
+apiVersion: apps/v1beta1
```

Comment:
Can you consider using `apps/v1beta2` or `apps/v1` for the API version. Both are available in all supported versions of Kubernetes by the community. This applies to all statefulsets in the PR.

Reply:
The `redis-ha` is able to run only on private custom image which image may not available in the future. The master name is also hardcoded in the image which name is `mymaster`.

Reply:
@KongZ Can you try a pull request to that chart to fix those issues?